docker-compose.yml

```diff
@@ -5,7 +5,7 @@ kafka:
   ports:
     - "9092:9092"
   environment:
-    KAFKA_VERSION: "0.10.1.0"
+    KAFKA_VERSION: "0.11.0.1"
```

Comment:
I double-checked and it looks like this environment variable isn't used anymore, we should use an explicit tag on the image instead.

Reply:
I think it's still in used : https://github.com/wurstmeister/kafka-docker/blob/master/Dockerfile#L8